### ISSUE TITLE
Loading the app does not periodically yield to allow rendering

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/Uno.Wasm.Bootstrap#771]
When the app is loading, the layouting of the browser is prevented as the thread is busy - so window size changes cause ugly white bars and the app seems frozen. It could be useful to break up the loading using `setTimeout` to allow the rendering to go through (if possible in terms of the current implementation)

![image](https://github.com/unoplatform/Uno.Wasm.Bootstrap/assets/1075116/e02ee13e-92ea-42c5-94ab-3f917a96dd0c)

**Comment by jeromelaban** [comment]
I don't think this is possible at this time, as the startup is mainly sequential and non-interruptible, but with threading we may be able to simplify this.
